GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/UIElementDropBehavior.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Interactivity;
+using System.Windows.Media;
+using System.Windows.Media.Effects;
+
+namespace GalaxyZooTouchTable.Behaviors
+{
+    public class UIElementDropBehavior : Behavior<UIElement>
+    {
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+
+            AssociatedObject.TouchEnter += AssociatedObject_TouchEnter;
+            AssociatedObject.TouchLeave += AssociatedObject_TouchLeave;
+        }
+
+        private void AssociatedObject_TouchLeave(object sender, TouchEventArgs e)
+        {
+            Border element = sender as Border;
+            element.Background = Brushes.Transparent;
+            element.Effect = null;
+
+            Border childBorder = element.Child as Border;
+            childBorder.Background = Brushes.Transparent;
+            childBorder.Effect = null;
+        }
+
+        private void AssociatedObject_TouchEnter(object sender, TouchEventArgs e)
+        {
+            Border element = sender as Border;
+            LinearGradientBrush brush = new LinearGradientBrush();
+            brush.StartPoint = new Point(0, 0.5);
+            brush.EndPoint = new Point(1, 0.5);
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#727272"), 0));
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#202020"), 1));
+            brush.GradientStops.Add(new GradientStop((Color)ColorConverter.ConvertFromString("#000000"), 1));
```

Comment:
Are the color constants here used throughout the app? It may at least be worth grabbing the color constant for black from the c sharp color library: https://docs.microsoft.com/en-us/dotnet/api/system.drawing.color.black?view=netframework-4.7.2#System_Drawing_Color_Black